src/scripts/main.js

```diff
@@ -1,7 +1,92 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const Game = require('../modules/Game.class');
+const fieldRow = [...document.querySelectorAll('.field-row')];
+const scoreCeil = document.querySelector('.game-score');
+const startBtn = document.querySelector('.start');
+const messageWin = document.querySelector('.message-win');
+const messageLose = document.querySelector('.message-lose');
+const messageStart = document.querySelector('.message-start');
 
-// Write your code here
+const game = new Game();
+
+const addField = () => {
+  if (game.getStatus() === 'win') {
+    messageWin.classList.remove('hidden');
+    document.removeEventListener('keydown', handleKeyDown);
+    game.gameStatus = 'playing';
+
+    return;
+  }
+
+  if (game.getStatus() === 'lose') {
+    messageLose.classList.remove('hidden');
+    document.removeEventListener('keydown', handleKeyDown);
+    game.gameStatus = 'playing';
+
+    return;
+  }
+
+  for (let i = 0; i < game.gameState.length; i++) {
+    const fieldCell = [...fieldRow[i].querySelectorAll('.field-cell')];
+
+    for (let j = 0; j < game.gameState[i].length; j++) {
+      const cell = fieldCell[j];
+      const cellValue = game.gameState[i][j];
+
+      if (cellValue === 0 && cell.textContent > 0) {
+        cell.classList.remove(`field-cell--${cell.textContent}`);
+        cell.textContent = '';
+      }
+
+      if (cellValue > 0) {
+        cell.classList.remove(`field-cell--${cell.textContent}`);
+        cell.classList.add(`field-cell--${cellValue}`);
+        cell.textContent = cellValue;
+      }
+    }
+  }
+  scoreCeil.textContent = game.gameScore;
+};
+
+const handleKeyDown = (e) => {
+  if (e.key === 'ArrowLeft') {
+    game.moveLeft();
+    addField();
+  }
+
+  if (e.key === 'ArrowRight') {
+    game.moveRight();
+    addField();
+  }
+
+  if (e.key === 'ArrowUp') {
+    game.moveUp();
+    addField();
+  }
+
+  if (e.key === 'ArrowDown') {
+    game.moveDown();
+    addField();
+  }
```

Comment:
you can simplify it using the switch case and move `addField();` out of cases and write only one time